onebusaway-android/src/main/res/drawable/zoom_btn_drop_shadow_left.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
```

Comment:
Could you please remove one of the new lines here (and elsewhere) so there is only one line in between elements?